src/client/entity-editor/common/entity-linked-search-field.js

```diff
@@ -101,6 +93,30 @@ function EntityLinkedSearchField(
 		...rest
 	}: EntityLinkedSearchFieldProps
 ) {
+	/**
+	 * Takes an entity and converts it to a format acceptable to react-select.
+	 *
+	 * @param {Object} entity the entity to convert
+	 * @returns {Object} the formatted data
+	 * @param {Array<LanguageOption>} languageOptions - The list of possible languages for an
+	 * entity.
+	 */
+
+	function entityToOption(entity) {
+		const id = isArea(entity) ? entity.id : entity.bbid;
+		const languageId = _.get(entity, ['defaultAlias', 'languageId']);
+		const languageIndex = languageOptions.find(index => index.value === languageId);
+		const language = languageOptions[languageIndex.value].label;
```

Comment:
And since that returns the object, you'll want to make sure you pass the label rather than the whole object (you did that correctly there, just a reminder to do so again)

Reply:
Yes, I consoled the output but didn't think to get the value directly form here. Sure, I will correct it.